controllers/util/storageconsumer.go

```diff
@@ -14,6 +14,7 @@ import (
 const (
 	// Reserved RadosNamespaceName for internal use and their representation at different layes
 	ImplicitRbdRadosNamespaceName = "<implicit>"
+	Is419AdjustedAnnotationKey    = "ocs.openshift.io/4_19-adjusted"
```

Comment:
Isn't this const already defined somewhere else in the codebase?

Reply:
yes, in another PR #3131

Reply:
rebased.